src/js/view/controls/components/button.ts

```diff
@@ -1,7 +1,8 @@
-import UI from 'utils/ui';
+import type UI from 'utils/ui';
 import svgParse from 'utils/svgParser';
 import helpers from 'utils/helpers';
 import type { PlayerAPI } from 'types/generic.type';
+import { addClickAction } from 'utils/add-click-action';
```

Comment:
Nitpick: nice to see all type imports at the bottom to separate imports that run vs just for type checking.